pkg/arvo/app/radar.hoon

```diff
@@ -0,0 +1,304 @@
+/+  *server
+|%
++$  move  [bone card]
++$  card
+  $%  [%poke wire dock pear]
+      [%wait wire @da]
+      [%rest wire @da]
+      [%info wire toro:clay]
+      [%connect wire binding:eyre term]
+      [%http-response =http-event:http]
+  ==
+::
++$  pear
+  $%  [%noun cord]
+      [%helm-hi cord]
+  ==
+::
++$  state
+  $:  scanning=_|                        :: are we currently scanning?
+      filter=(set ship)
+      configured=(set ship)              :: set of ships to scan
+      outstanding=(map ship @da)         :: ships we've |hi'd with no reply
+      log=(map ship (list [@da @da]))    :: log of past |hi's sent/recv by ship
+      timers=(map wire @da)              :: map of delay timers
+      moves=(list move)                  :: list of outgoing moves
```

Comment:
Note that if you do this, you must still reinitialize `moves` to null, since our context is only cleared when you reload the app (eg change its source).

Reply:
Oh, right.  I still think they should be bunted on each app invocation instead of needing a manual reload.  Not sure how other apps do this

Reply:
The "common pattern" I was referring to is really in vanes, not apps, because there's a good opportunity to do this in between the two vane activation steps.

Reply:
Three options, none of them great:

- an explicit `+abet`, as here
- a separate core we create each time that tracks its own moves (most common, but also the most overhead unless you want that core for another reason)
- just handling the moves directly, which gives lots of

```
=^  moves-1  this  foo
=^  moves-2  this  bar
[(weld moves-1 moves-2) this]
```